### ISSUE TITLE
MTContext failed with iOS 16

### ISSUE THREAD

**Issue by adirkol** [MetalPetal/MetalPetal#332]
tldr:
When creating MTIContext with the new iOS 16.0, it returns an error: "Compiler failed to build request"

### Checklist
- [ X] I've read the [README](https://github.com/MetalPetal/MetalPetal/blob/master/README.md)
- [ X] If possible, I've reproduced the issue using the `master` branch of this repo
- [ X] I've searched for [existing GitHub issues](https://github.com/MetalPetal/MetalPetal/issues)

### Environment

 Info                    | Value                               |
-------------------------|-------------------------------------|
 MetalPetal Version      | e.g. 1.24.2
 Integration Method      | e.g. SwiftPM
 Platform & Version      | e.g. iOS 16.0
 Device                  | e.g. MacBook Pro 15' / iPhone 12 Pro

### Steps to Reproduce
Here's the code I'm using, which worked before iOS 16.0:
```
if let device = MTLCreateSystemDefaultDevice() {
    let options = MTIContextOptions()
    self.context = try? MTIContext(device: device, options: options)
}
```
Seems like options and device is being created properly, but after trying to create the MTIContext I get this error in the debugger: "Compiler failed to build request", and obviously self.context is nil.

### Expected behavior
MTIContext should return a context.

### Actual behavior
MTIContext returns nil

**Comment by adirkol** [comment]
Seems like removing the package and re-add it solves the problem.
